src/blocks/scratch3_sensing.js

```diff
@@ -124,4 +125,47 @@ Scratch3SensingBlocks.prototype.daysSince2000 = function () {
     return mSecsSinceStart / msPerDay;
 };
 
+Scratch3SensingBlocks.prototype.getAttributeOf = function (args, util) {
+    var attrTarget;
+
+    if (args.OBJECT === '_stage_') {
+        attrTarget = this.runtime.getTargetForStage();
+    } else {
+        attrTarget = this.runtime.getSpriteTargetByName(args.OBJECT);
+    }
+
+    // Generic attributes
+    if (attrTarget.isStage) {
+        switch (args.PROPERTY) {
+        // Scratch 1.4 support
+        case 'background #': return attrTarget.currentCostume + 1;
+
```

Comment:
I'm not sure I like this whitespace, but keeping every line under 80 characters is difficult..